### ISSUE TITLE
Tesseract is not working properly.

### ISSUE THREAD

**Issue by nijanthan0** [tesseract-ocr/tesseract#2384]
![out3](https://user-images.githubusercontent.com/46311492/55938931-1e7f9300-5c5a-11e9-96bc-a9c0c407791d.png)
![out3](https://user-images.githubusercontent.com/46311492/55938947-2808fb00-5c5a-11e9-98c0-534f060d0af2.png)
![out3](https://user-images.githubusercontent.com/46311492/55938998-4e2e9b00-5c5a-11e9-9a44-b961ef5811c4.png)
[ac036001.pdf](https://github.com/tesseract-ocr/tesseract/files/3067440/ac036001.pdf)

Before you submit an issue, please review [the guidelines for this repository](https://github.com/tesseract-ocr/tesseract/blob/master/CONTRIBUTING.md).

Please report an issue only for a BUG, not for asking questions.

Note that it will be much easier for us to fix the issue if a test case that
reproduces the problem is provided. Ideally this test case should not have any
external dependencies. Provide a copy of the image or link to files for the test case.

Please delete this text and fill in the template below.

------------------------

### Environment

* **Tesseract Version**: 
* **Commit Number**: 
* **Platform**: 

### Suggested Fix:
![out2](https://user-images.githubusercontent.com/46311492/55939049-6acad300-5c5a-11e9-85f8-5bca774262b8.png)
the image file contains Tamil,English,Numbers.
Tamil words extracted accurately but numbers in the image cannot.. 
please tell me some solution

**Comment by Shreeshrii** [comment]
If you don't give the correct command how will you get the correct result. Tesseract is working properly.

If you want English to be recognized in addition to Tamil, OCR using the two languages together by `-l tam+eng`.

```
சட்டமன்றத்‌ தொகுதி எண்‌ மற்றும்‌ பெயர்‌ : 36-உத்திரமேரூர்‌                                                   பாகம்‌ எண்‌ : 36

பிரிவு எண்‌ மற்றும்‌ பெயர்‌ : 1-முத்தியால்பேட்டை (ஊ), ஏரிவாய்‌ (வ.கி), வார்டு 4 ஏரிவாய்‌

_ 1                        1813463007 3?                        KBT3462801                  TN/05/025/0165728

பெயர்‌: லட்சுமி -                              பெயர்‌: அன்னம்மாள்‌ -                       பெயர்‌: மல்லிகா -

கணவர்‌ பெயர்‌: நாராயணசாமி -              கணவர்‌ பெயர்‌: சுப்பிரமணி -                 கணவர்‌ பெயர்‌: செல்வம்‌ -

வீட்டு எண்‌: 1                        Photo is     வீட்டு எண்‌: *                        Photo is     வீட்டு எண்‌: *                         Photo is

வயது: 71 பாலினம்‌ :பெண்‌        Available |] | வயது: 66 பாலினம்‌ :பெண்‌        Available | | வயது: 88 பாலினம்‌ :பெண்‌        Available
```

Please close this issue.

**Comment by nijanthan0** [comment]
Yes,I know but numbers are not properly extracted. That's my issue.

**Comment by Shreeshrii** [comment]
Where is the requested info for filing an issue? What command did you use?

Environment
Tesseract Version:
Commit Number:
Platform:

**Comment by nijanthan0** [comment]
tesseract out2.png out2 -l tam+eng --psm 6

**Comment by nijanthan0** [comment]
tam.traineddata
tesseract version:4.1.0-rc1-125-gac7e

**Comment by nijanthan0** [comment]
[0ut2.txt](https://github.com/tesseract-ocr/tesseract/files/3067824/0ut2.txt)
My output text file.

**Comment by Shreeshrii** [comment]
>tam.traineddata

from - tessdata, tessdata_fast, tessdata_best ?

**Comment by nijanthan0** [comment]
tessdata_best

**Comment by Shreeshrii** [comment]
>numbers are not properly extracted. That's my issue.

Which numbers?

Either provide a `ground truth` for the page or give examples.

**Comment by nijanthan0** [comment]
![out2](https://user-images.githubusercontent.com/46311492/55946718-d0bf5680-5c6a-11e9-922b-971ee443a2e8.png)
this is the input image

![mistakes](https://user-images.githubusercontent.com/46311492/55946577-99e94080-5c6a-11e9-9729-57be15e1b2c8.png)

The above of highlighted words are both English and Numbers. Both of them not extracted properly.

**Comment by Shreeshrii** [comment]
Try with

tessdata_best/script/Tamil.traineddata

**Comment by nijanthan0** [comment]
I tried it numbers are extracted correctly but some numbers missed in the text.

**Comment by nijanthan0** [comment]
![mis](https://user-images.githubusercontent.com/46311492/55958324-f73ebb00-5c85-11e9-94b0-4bbacf5f1e74.png)

**Comment by gnana70** [comment]
Please check this repo if you are interested in tamil ocr. It's capable of recognizing both tamil and English. https://github.com/gnana70/tamil_ocr

You can simply install using ```pip install ocr_tamil```

```python
from ocr_tamil.ocr import OCR
image_path = r"test_images\0.jpg" # insert your own path here
ocr = OCR(detect=True)
texts = ocr.predict(image_path)
with open("output.txt","w",encoding="utf-8") as f:
    f.write(texts)

>>>> கொடைக்கானல் Kodaikanal 

```
